docker-compose.yml

```diff
@@ -94,12 +94,13 @@ services:
   addons-frontend:
     <<: *env
     environment:
+      - HOSTNAME=uitests
```

Comment:
Why is this needed?

Reply:
For the node_config package that loads node configs on addons-frontend.